src/components/Model.tsx

```diff
@@ -0,0 +1,28 @@
+export interface Block {
```

Comment:
this file shouldn't have tsx extension, it should only be .ts since you dont have html and css in it.
Also it should be in the folder model instead of components